src/wp-content/themes/classicpress-twentyfifteen/functions.php

```diff
@@ -26,10 +26,13 @@ function cp2015_enqueue_styles() {
     $parent_style = 'twentyfifteen-style';
 
     wp_enqueue_style( $parent_style, get_template_directory_uri() . '/style.css' );
+	
+    $version = classicpress_asset_version( 'style', 'classicpress-twentyfifteen' );
     wp_enqueue_style( 'classicpress-twentyfifteen',
         get_stylesheet_directory_uri() . '/style.css',
         array( $parent_style ),
-        filemtime( get_stylesheet_directory() . '/style.css' )
+	$version
+        //filemtime( get_stylesheet_directory() . '/style.css' )
```

Comment:
Thanks for the PR.  The call to `classicpress_asset_version` is correct, however there are some things that need to be fixed before it can be merged.

1. Please don't leave this comment in place here, if we need to see what this was before then we can use the `git` history.
2. I don't think assigning to a `$version` variable is necessary.  Why not just put the `classicpress_asset_version` in that same place instead?
3. This same change needs to be made a total of three times, in `classicpress-twentyfifteen`, `classicpress-twentysixteen`, and `twentyseventeen`.

Reply:
And 4. please use tab characters instead of spaces for indentation.

Reply:
fixes committed and classicpress_asset_version is now also applied to other child themes.